source/test-suite/Main.hs

```diff
@@ -1459,13 +1476,13 @@ expectDiscover files input expected = do
 runGild ::
   [String] ->
   [(Input.Input, ByteString.ByteString)] ->
-  [(FilePath, [FilePath])] ->
+  [[String]] ->
   (Either E (), S, W)
 runGild arguments inputs files =
   runTest
     (Gild.mainWith arguments)
     ( Map.fromList inputs,
-      Map.mapWithKey (\d fs -> FilePath.combine d <$> fs) $ Map.fromList files
+      Map.singleton "." (fmap FilePath.joinPath files)
```

Comment:
This changed since now `walk` is only ever called with `"."` as the first argument.